xml/art_amd-sev.xml

```diff
@@ -21,14 +21,19 @@
   <abstract>
    <para>
     AMD's Secure Encrypted Virtualization (SEV) allows the memory of virtual
-    machines to be encrypted. This is a new feature for Linux's built-in
-    Kernel-based Virtual Machine (&kvm;) hypervisor. The intention is to
-    increase system security, especially when using persistent memory.
+    machines to be encrypted. SEV with Encrypted State (SEV-ES) goes one step
+    further by encrypting the virtual machine's CPU register content. These
+    technologies increase system security and are ideal for multi-tenant
+    environments such as cloud computing. They enable protection from a variety
+    of cross-VM and hypervisor-based attacks. The growing use of persistent
```

Comment:
The mentioning of persistent memory can be removed entirely from this document, as it is misleading in SEV context. Encrypted data stored on persistent memory will be lost after the VM is destroyed, because the key used for encryption is also destroyed with the VM.

Reply:
I added another commit to remove the bit about persistent memory and reword the affected paragraphs. Before adding it to this PR, I'd like to know if the scenario described in the introduction is actually possible. I.e, if a persistent memory module used by an unencrypted VM was pulled from the system, could it be placed in another system and the unencrypted contents read? If so, SEV does help right, since the contents would be encrypted.

Reply:
@joergroedel can you please answer my question above? I'm happy to send a new PR to remove the persistent memory stuff, but I'm curious if the existing scenario is actually possible. Thanks!